examples/time_series/MvGaussianRandomWalk_demo.ipynb

```diff
@@ -2,6 +2,7 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    α_mean = trace.posterior["α"].mean(axis=(0, 1))
</pre><p>rather than <code>axis=(0, 1)</code></p><p>which is a bit mysterious, would <code>.mean(dim=('chain', 'sample'))</code> work?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Used `dim=("chain", "draw")`.